### ISSUE TITLE
ERROR: Invalid session when trying to fetch trade confirmations.

### ISSUE THREAD

**Issue by doktokto** [Jessecar96/SteamBot#1061]
I receive this error. Only after restart bot can accept offers.

**Comment by doktokto** [comment]
Now I am receiving this error all the time.

**Comment by Noobgam** [comment]
#1048 explains enough, you didn't need to reopen new issue.
